modules/sway/hm.nix

```diff
@@ -21,7 +21,44 @@ in {
     (lib.mkIf config.stylix.targets.sway.enable {
       wayland.windowManager.sway.config = {
         inherit fonts;
+          startup =
+          let
+            slideshowScript = pkgs.writeScript "script.sh" ''
```

Comment:
This slideshow script is repeated in a few places, I've only reviewed this copy, maybe we should move it to a central file if possible

Reply:
i think moving the wayland one to lib would be a good idea cause that can be used in other compositors like river, however the xorg version i can only really see working through xsession.